app/frontend/src/views/Submission.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <v-container>
+    <BaseSecure admin>
+      <h2>Show the form data here</h2>
+      <v-container>
+        <!-- <v-row>
+          <v-col offset-lg="1" cols="12" lg="10">
+            <v-card outlined class="review-form">
+              <h2 class="review-heading">
+                1. Before You Begin
+                <v-btn color="primary" class="mx-5" fab x-small @click="setStep(1)">
+                  <v-icon>mdi-pencil</v-icon>
+                </v-btn>
+              </h2>
+              <Step1 />
+            </v-card>
+
+            <v-card outlined class="review-form">
+              <h2 class="review-heading">
+                2. Contact Information
+                <v-btn color="primary" class="mx-5" fab x-small @click="setStep(2)">
+                  <v-icon>mdi-pencil</v-icon>
+                </v-btn>
+              </h2>
+              <Step2 reviewMode="true" />
+            </v-card>
+
+            <v-card outlined class="review-form">
+              <h2 class="review-heading">
+                3. Before Workers Arrive
+                <v-btn color="primary" class="mx-5" fab x-small @click="setStep(3)">
+                  <v-icon>mdi-pencil</v-icon>
+                </v-btn>
+              </h2>
+              <Step3 reviewMode="true" />
+            </v-card>
+
+            <v-card outlined class="review-form">
+              <h2 class="review-heading">
+                4. After Workers Arrive
+                <v-btn color="primary" class="mx-5" fab x-small @click="setStep(4)">
+                  <v-icon>mdi-pencil</v-icon>
+                </v-btn>
+              </h2>
+              <Step4 reviewMode="true" />
+            </v-card>
+
+            <v-card outlined class="review-form">
+              <h2 class="review-heading">
+                5. If Workers Become Ill
+                <v-btn color="primary" class="mx-5" fab x-small @click="setStep(5)">
+                  <v-icon>mdi-pencil</v-icon>
+                </v-btn>
+              </h2>
+              <Step5 reviewMode="true" />
+            </v-card>
+          </v-col>
+        </v-row>-->
+      </v-container>
+    </BaseSecure>
+  </v-container>
+</template>
+
+<script>
+import ipcService from '@/services/ipcService';
+
+// import { mapActions, mapGetters, mapMutations } from 'vuex';
+
+// import Step1 from '@/components/form/Step1.vue';
+// import Step2 from '@/components/form/Step2.vue';
+// import Step3 from '@/components/form/Step3.vue';
+// import Step4 from '@/components/form/Step4.vue';
+// import Step5 from '@/components/form/Step5.vue';
+
+export default {
+  name: 'Submission',
+  components: {
+    // Step1,
+    // Step2,
+    // Step3,
+    // Step4,
+    // Step5,
+
+  },
+  data() {
+    return {
+      ipcPlanId: ''
+    };
+  },
+  methods: {
+    // get table data from frontend service layer
+    getData() {
+      ipcService
+        .getIPCContent('7315ba58-fb9d-4e88-8896-22180763355b')
```

Comment:
this is where i send the ipcPlanId to the backend. I was testing with a value from my database.
the console.log shows me i am getting all form submissions

Reply:
In order for any of those components to render they'll need the state updated (they pull all fields computed from the Vuex store).
So this isn't where the call to get the form would be. It should dispatch an action on mounted that gets the form and then mutates the vuex store to the state it needs.

Reply:
ok. thanks. Appreciate the input. to be honest i'm not sure i understand this. i'll try and take another look at this tonight

Reply:
Ok no prob. I'll give it a think too. TBH I'm not 100% on how it's all going to fit together fetching store state on the admin side. I'm fairly certain we just need an action to fetch a specific form by ID as you've got and then set the existing objects in the store with those results via the existing mutations.
I'm a bit concerned about what to do with the store when switching from one form to another so I think we'll need a way to clear it as well, or rock-solid error handling.
Might need to switch over to dynamic loading the same way we do in some modules in GETOK